CHANGELOG.rst

```diff
@@ -1,3 +1,22 @@
+=======================================
+SAP Operations Collection Release Notes
+=======================================
+
+.. contents:: Topics
+
+v1.0.0
+======
+
+Major Changes
+-------------
+
+- SAP kernel update role.
+- parameter_info - module to fetch parameter information.
+- service - module to manage SAP HANA services.
+- system_info module - module to fetch SAP system information.
+- system module - module to manage SAP system.
+
+
```

Comment:
changelogs should be provided as a fragment here:
https://github.com/sap-linuxlab/community.sap_operations/tree/main/changelogs/fragments

and then generated 
https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_release_without_branches.html

Reply:
We do not use release branches, we release from main. 
We are not yet so far with automatic CHANGELOG.rst generation so this task is still manual for us. 
But thank for pointing out to documentation. We will definitely automate this in the future.

Reply:
you can even do it manually because the changelog you provided is wong formatted